Sources/TokamakCore/Props/Font.swift

```diff
@@ -0,0 +1,167 @@
+// Copyright 2018-2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+public struct Font: Hashable {
```

Comment:
`danger-lint` is still failing (I've updated the token environment variable again in `main`, fingers crossed that finally works in future PRs), but if it worked it would complain about line length. I think it's ok to keep long lines in this specific file as splitting them is repetitive, just need to silence the warning:

```suggestion
// swiftlint:disable line_length
public struct Font: Hashable {
```